### PR TITLE
Update jquery-validation-unobtrusive

### PR DESCRIPTION
**Pull request by jbagga**
Addresses https://github.com/aspnet/templating/issues/15

Update jquery-validation and jquery-validation-unobtrusive
Added a LICENSE file for jquery-validation-unobtrusive
Updates the templates baselines
Manually tested template creation and the general scenarios

### CODE REVIEW

**Comment by jbagga** [comment]
cc @mkArtakMSFT

**Comment by jbagga** [comment]
https://github.com/aspnet/templating/issues/422 to track removing bower going forward

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/lib/jquery-validation/.bower.json [hunk 1]
```diff
@@ -27,14 +27,15 @@
   "dependencies": {
     "jquery": ">= 1.7.2"
   },
-  "version": "1.14.0",
-  "_release": "1.14.0",
+  "version": "1.17.0",
```
Comment (kichalla) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/lib/jquery-validation/.bower.json]: I think we need to match these versions in the following file too:
https://github.com/aspnet/templating/blob/release/2.1/src/THIRD-PARTY-NOTICES#L170
cc @Eilon

Comment (jbagga) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/lib/jquery-validation/.bower.json]: Done

**Pull request merged**